code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -445,6 +445,10 @@
 	name = "spellbook of [spellname]" //Note, desc doesn't change by design
 	..()
 
+/obj/item/weapon/spellbook/oneuse/mindswap/recoil(var/mob/user)
+	qdel(src)
+
+/*
```

Comment:
Commenting out code with block comments fucks with version control
That's how we got MEGAMADCRAB

Reply:
Fine